### PR TITLE
Set 1800s timeout for resilience modules tests

### PR DESCRIPTION
**Pull request by KVS85**
http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-matrix-only-grey/794/

### CODE REVIEW

File: js/client/modules/@arangodb/testsuites/resilience.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ var _resilience = function(path) {
     let testCases = tu.scanTestPaths(testPaths[path], options);
```
Comment (dothebart) [inline js/client/modules/@arangodb/testsuites/resilience.js]: please `let localOptions = _.clone(options)` and use that later on.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8847/

http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-matrix-only-grey/795/

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8850/

http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-matrix-only-grey/797/

**Comment by KVS85** [comment]
Tests blue except expected known failures.

**Pull request merged**